src/lgdo/lh5/store.py

```diff
@@ -199,10 +285,24 @@ def write(
         # write_object:overwrite.
         mode = "w" if wo_mode == "of" else "a"
 
+        file_kwargs = {
+            k: h5py_kwargs[k]
+            for k in h5py_kwargs & signature(h5py.File).parameters.keys()
+        }
+        if page_buffer:
+            file_kwargs.update(
+                {
+                    "fs_strategy": "page",
+                    "fs_page_size": page_buffer,
+                    "fs_persist": True,
+                    "fs_threshold": 1,
+                }
+            )
```

Comment:
This code is duplicated in this file and in `core.py`, can it be defined in only one place?

Reply:
Back in this pull request https://github.com/legend-exp/legend-pydataobj/pull/97, I moved a lot of the code that opened files and found the group being read in into `store.read` and `core.read`, which created a lot of redundancy. I think both of these functions should be revisited. I actually tried fixing this at some point by having `store.read` use `core.read`, but it caused an error that I didn't have time to sort through; as I recall it had to do with how the output is sometimes an lgdo object and some times a tuple.